APMrover2/sailboat.cpp

```diff
@@ -134,6 +137,9 @@ void Sailboat::init()
     // initialise motor state to USE_MOTOR_ASSIST
     // this will silently fail if there is no motor attached
     set_motor_state(UseMotor::USE_MOTOR_ASSIST, false);
+
+    // set the filter freq, use constant for time being, filter is updated at 400hz
+    tack_filt.set_cutoff_frequency(400.0f, SAILBOAT_TACK_FILT);
```

Comment:
I wonder if this "400.0f" is meant to be the main loop rate?  In which case it's actually 50hz by default for Rovers but we can also use scheduler.get_loop_rate_hz() I think.

Reply:
it did seem higher than I expected, i got the 400 from here https://github.com/ArduPilot/ardupilot/blob/8736a13a875fbc9381795ed06c24bb72d44d96d6/APMrover2/Rover.cpp#L52 , not that it matters if were moving it

Reply:
ok, so that rate is the maximum it will ever run at but it won't achieve that if the main loop is set to run slower.  Let's replace the 400 with the get-loop-rate-hz call.